<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>621</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ChooseOutputButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
  <widget class="QLabel" name="filenametext">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Uploaded File: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload File</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImageShowLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="threshold_slider">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>140</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="ImageScrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>721</width>
     <height>141</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>719</width>
      <height>123</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>701</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>602</x>
     <y>520</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Selected Images</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detection_start">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Detection</string>
   </property>
  </widget>
  <widget class="QLabel" name="threshold_text">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold: 0.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_text">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output directory: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="youtubepage_button">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>520</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Change to Youtube Dowonload -&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="slides_selected">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Slides selected: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
